### ISSUE TITLE
Advanced Customization: Edit mobile tab bar colors changes home page colors

### ISSUE THREAD

**Issue by onaseef** [onaseef/yomobi#333]
I don't know how to reproduce this bug, but it has happened numerous times.   Somehow, modifying the mobile tab bar background color changes the home page background color instead.

I think it's just the background color control and perhaps the behavior is that the background controller doesn't update because it's also happened where it updated the header background when I was on the mobile tab bar section.

Is it possible the controls have the same ID? Or when changes are made and then cancelled by clicking on a different widget, the values aren't reset?

I'm just trying to brainstorm why this might happen. If you would look at the code and see if there's something that isn't being set or reset properly that might explain this.

**Comment by gilbert** [comment]
In your first case, do you mean changing the mobile tab bar background color updates the home page background color in real time, or "resets" it to a different color?

Each color picker has a predefined target area to change, so I can't think of why this is happening other than the one you're changing is different than what you think, i.e. the wrong one is showing up somehow. I played around with it a bunch and couldn't get the bug to appear.

If you're in chrome, you can right click the color picker and "Inspect Element". A DOM tree will pop up. You'll see something like this: http://ge.tt/9zrmL2J/v/0

In that screen shot, you can see the div with `class="color-picker"` and, more importantly, `data-target="icon_text_color"`. The next time you see this happen, make sure the `data-target` is consistent with what it's changing. If it is, then the wrong color picker is somehow showing up (maybe the wrong tab content). If it's not... I'm not sure.

**Comment by onaseef** [comment]
Answer: In the first case, the mobile tab background color updates the home page background color in real time.

**Comment by onaseef** [comment]
Now the home page icon color picker is modifying the header text color. I did the check you recommended and even though the class="color-picker" and the data-target="icon_text_color", the wrong element is being modified.

See: http://ge.tt/3f9JnrJ/v/0

**Comment by gilbert** [comment]
I played with this for 10 minutes on ie8, but I still can't reproduce it. The code is very straightforward, pulling data directly from the dom, so I'm kind of at a loss.

I guess I have another question. When you click outside the color picker to close it, and then click to open it again, does the problem persist?

**Comment by onaseef** [comment]
Yes, the problem persists until I do a browser refresh.  Even if I select a different element from the dropdown to edit, it persists.  My thought was maybe the target was being stored somewhere in a variable and not being updated?

**Comment by gilbert** [comment]
Nope, there aren't any persistent variables interacting with the color picker, which is why I'm so confused about this.
